01 Operator Overloading/Operator Overloading.Host/Program.cs

```diff
@@ -13,28 +14,26 @@ static void Main(string[] args)
         {
             try
             {
-                Console.WriteLine("Enter 1st Amount : ");
-
+                Console.WriteLine("Format : 100 USD");
+                Console.WriteLine("\nEnter Amount : "); // Enter Amount in Format '100 USD'
                 var m1 = new Money(Console.ReadLine());
 
+                Console.WriteLine("\nEnter target Currency : "); // Enter target currency
+                var m2 = new Money(0, Console.ReadLine());
```

Comment:
I've created money object here because i want to pass target currency's value to 'Currency' property for validation. 
If i use
var output = m1.Convert(Console.ReadLine());
Then i have to create separate validation for target currency in Convert() function.